dhall-lsp-server/dhall-lsp-server.cabal

```diff
@@ -46,22 +46,22 @@ library
     , aeson-pretty         >= 0.8.7    && < 0.9
     , base                 >= 4.11     && < 5
     , bytestring           >= 0.10.8.2 && < 0.12
+    , co-log-core          >= 0.3.1.0  && < 0.4
     , containers           >= 0.5.11.0 && < 0.7
     , data-default         >= 0.7.1.1  && < 0.8
     , directory            >= 1.2.2.0  && < 1.4
     , dhall                >= 1.38.0   && < 1.43
     , dhall-json           >= 1.4      && < 1.8
     , filepath             >= 1.4.2    && < 1.5
-    , lsp                  >= 1.2.0.0  && < 1.5
-    , rope-utf16-splay     >= 0.3.1.0  && < 0.5
-    , hslogger             >= 1.2.10   && < 1.4
+    , lsp                  >= 1.5.0.0  && < 2
     , lens                 >= 4.16.1   && < 5.2
```

Comment:
Would you bump the upper bound on lens, please, too, if it is safe to do so — this is causing breakage in stackage and thus also nixpkgs. The tool does build without the upper bound (I haven't yet tested it).

Reply:
It is safe to do so; I bumped the upper bounds.